administration/src/application/components/ApplyController.tsx

```diff
@@ -10,14 +10,16 @@ import useLocallyStoredState from '../useLocallyStoredState'
 import DiscardAllInputsButton from './DiscardAllInputsButton'
 import { useGarbageCollectArrayBuffers, useInitializeGlobalArrayBuffersManager } from '../globalArrayBuffersManager'
 import ApplicationForm from './forms/ApplicationForm'
-import { useMemo } from 'react'
+import { useMemo, useState } from 'react'
 import { SnackbarProvider, useSnackbar } from 'notistack'
+import BasicDialog from '../../components/BasicDialog'
```

Comment:
Please move this file somewhere under src/application. We don't want to use "share" components between the application form and the digitale druckerei